bank-struct-queries/bank-struct/SocialStatus.sql

```diff
@@ -0,0 +1,13 @@
+﻿create table [bank-task].dbo.Social_Status(
```

Comment:
Naming should be same for all tables, if all tables named by plural entities, it have to be Social_Statuses